policy-service/src/main/java/uk/gov/gchq/palisade/service/policy/config/PolicyConfiguration.java

```diff
@@ -29,7 +29,8 @@
      *
      * @return a {@link List} of the objects that have implemented {@link PolicyPrepopulationFactory}.
      */
-    @SuppressWarnings("java:S1452") // Allow generics wildcard
+    @SuppressWarnings("java:S1452")
+    // Allow generics wildcard
```

Comment:
```suggestion
    // Allow generics wildcard
    @SuppressWarnings("java:S1452")
```
Should the comment come before the suppression?